plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -90,6 +93,12 @@
   private static final Joiner MISSING_GROUP_MEMBER_JOINER = Joiner.on(", "); //$NON-NLS-1$
 
   private static final String ARGUMENTS_SEPARATOR = "="; //$NON-NLS-1$
+  private static Image pipelineImage;
+  static {
+    ImageDescriptor imageDescriptor = AbstractUIPlugin
+        .imageDescriptorFromPlugin(DataflowUiPlugin.PLUGIN_ID, "icons/Dataflow_16.png");
+    pipelineImage = imageDescriptor != null ? imageDescriptor.createImage() : null;
```

Comment:
This still leaks the item such as after the class is GC'd.  Just allocate and dispose of this image on a per-tab basis.